lib/business_time/business_days.rb

```diff
@@ -1,15 +1,17 @@
 require 'active_support/time'
+require 'currency'
```

Comment:
Instead of multiple requires everywhere, just require it in topmost `business_time.rb` before everything.